ArchiSteamFarm/Localization/Strings.resx

```diff
@@ -668,4 +668,8 @@ StackTrace:
   <data name="BotHasNoWallet" xml:space="preserve">
     <value>Bot has no wallet.</value>
   </data>
-</root>
+  <data name="BotLevel" xml:space="preserve">
+    <value>{0} level.</value>
+    <comment>{0} will be replaced by account level</comment>
```

Comment:
account -> account's and period on the end.

Reply:
```
{
  "Result": "<a> has 1 level.\r\n<b> has 2 level.\r\n<c> has 5 level.",
  "Message": "OK",
  "Success": true
}
```
it's ok?

Reply:
You can't assume that this string will always be together with bot name, so no, this is invalid. Use `Bot has level {0}.` which works always.